core/server/master/src/main/java/alluxio/master/file/InodeSyncStream.java

```diff
@@ -452,9 +453,10 @@ private SyncStatus syncInternal() throws
        * Note that this still applies if A is to sync recursively path /aaa while B is to
        * sync path /aaa/bbb as the sync scope of A covers B's.
        */
-      boolean shouldSync = mUfsSyncPathCache.shouldSyncPath(mRootScheme.getPath(), mSyncInterval,
+      boolean shouldSkipSync =
+          mUfsSyncPathCache.shouldSyncPath(mRootScheme.getPath(), mSyncInterval,
           mDescendantType).getLastSyncTime() > mRootScheme.shouldSync().getLastSyncTime();
-      if (shouldSync) {
+      if (shouldSkipSync) {
```

Comment:
shouldSkipSync and shouldSync are totally different meaning. Is it a bug previously?

Reply:
yes the previous naming was wrong this PR fixed that